stock_demand_estimate/models/stock_demand_estimate.py

```diff
@@ -91,20 +94,3 @@ def name_get(self):
             )
             res.append((rec.id, name))
         return res
-
-    @api.model
-    def get_quantity_by_date_range(self, date_start, date_end):
```

Comment:
Do not remove this, it is meant to be used in other modules as for example DDMRP

Reply:
Ok, I put a docstring.